src/components/BookList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Book } from './Book';
+
+export class BookList extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  render() {
+    const { books } = this.props;
+    return (
+      books.map((book) => <Book key={book.id} book={book} />)
+    );
+  }
+}
```

Comment:
- [ ] Your app runs with errors. I'm not able to see the home page. Your app shows only a white page after running the `npm run start`. Please ensure that your app works as expected and runs without errors.

<img width="218" alt="image" src="https://user-images.githubusercontent.com/77190203/179044797-3c582477-2936-4fb0-baa3-66a435fb7cea.png">

Reply:
Hello there @medaminedev66 Thank you for the review, however, I do not see these errors on my side, do you know why that would be? 

All the best-K